plugin/db/driver.go

```diff
@@ -35,6 +35,10 @@ const (
 	BytebaseDatabase = "bytebase"
 )
 
+func (t Type) String() string {
```

Comment:
For consistency with other custom types to string conversions, we do not add a `String()` method, and just use `string(xxx)` in place.